playground/graph/graph.ml

```diff
@@ -0,0 +1,118 @@
+module Node : sig
+  type elt = int
+  type t
+  val create : elt -> t
+  val value : t -> elt
+  val compare : t -> t -> int
+  val id : t -> int
+  val to_string : t -> string
+end = struct
+  type elt = int
+  type t = { id : int; value : elt }
+  
+  let counter = ref 0
+
+  let create v = 
+    let id = !counter in
+    incr counter;
+    { id = id; value = v }
+
+  let value node = node.value
+
+  let id node = node.id
```

Comment:
Ali to funkcijo rabimo? Po mojem je bolje, da je id skrit.

Reply:
379652b20a5bc49e5fbc44c43ab795fafadd67ed